include/nlohmann/json.hpp

```diff
@@ -2224,6 +2238,14 @@ class basic_json // NOLINT(cppcoreguidelines-special-member-functions,hicpp-spec
         return value(ptr, string_t(default_value));
     }
 
+    template<typename BasicJsonType>
+    JSON_HEDLEY_DEPRECATED(3.10.6)
```

Comment:
Please use `JSON_HEDLEY_DEPRECATED_FOR` and describe what to use instead (see https://nemequ.github.io/hedley/api-reference.html#HEDLEY_DEPRECATED_FOR).